diff -urN mod_suid2-0.3.org/mod_suid2.c mod_suid2-0.3/mod_suid2.c
--- mod_suid2-0.3.org/mod_suid2.c	2004-09-01 00:41:53.799114624 +0200
+++ mod_suid2-0.3/mod_suid2.c	2004-09-01 00:43:40.123950792 +0200
@@ -180,10 +180,10 @@
 #include "mpm_common.h"
 
 #define MODULE                "mod_suid2"
-#define SUID_DEFAULT_USER     "nobody"
-#define SUID_DEFAULT_GROUP    "nobody"
-#define UID_MIN               100
-#define GID_MIN               100
+#define SUID_DEFAULT_USER     "http"
+#define SUID_DEFAULT_GROUP    "http"
+#define UID_MIN               1000
+#define GID_MIN               1000
 #define _NDEBUG	              1
 
 typedef struct
