diff -urN mod_suid2-0.2.1.org/mod_suid2.c mod_suid2-0.2.1/mod_suid2.c
--- mod_suid2-0.2.1.org/mod_suid2.c	2004-03-27 02:15:52.396826096 +0100
+++ mod_suid2-0.2.1/mod_suid2.c	2004-03-27 02:18:43.577802632 +0100
@@ -168,10 +168,10 @@
 #include "mpm_common.h"
 
 #define MODULE                "mod_suid2"
-#define SUID_DEFAULT_USER     "nobody"
-#define SUID_DEFAULT_GROUP    "nobody"
-#define UID_MIN               100
-#define GID_MIN               100
+#define SUID_DEFAULT_USER     "http"
+#define SUID_DEFAULT_GROUP    "http"
+#define UID_MIN               1000
+#define GID_MIN               1000
 #define NDEBUG	              1
 
 typedef struct
